README.md

```diff
@@ -1939,6 +1923,26 @@ can verify in the [demo app](https://fir-ui-demo-84a6c.firebaseapp.com/)).
 
 **For v1.0.0 and superior:** https://github.com/firebase/firebaseui-web/releases
 
+### 6.0.0
+
```

Comment:
I think it might be better to add an intro under the line below, to let developers understand their when they can choose from using Firebase V8, V9 compat, V9 and what to do along with using FirebaseUI.

`FirebaseUI-web v6.0.0 is intended to be used alongside Firebase JS SDK v9 compat.`
Something like:
We are migrating FirebaseUI to use [Firebase JS SDK v9](https://firebase.google.com/support/release-notes/js). There will be two phases of migration, we’re currently at phase 1 for v6.0.0:
1. Change to Firebase dependency v9 by importing Firebase compat library without changing code.
2. Change to modularized import which will be a breaking change, this will happen shortly after the GA launch of Firebase JS SDK v9.

For those developers:
1. For those who want to still use FirebaseUI with Firebase V8, you can keep import unchanged for now
before the phase 2 migration to Firebase JS SDK v9.
2. For those who want to use FirebaseUI with Firebase V9, you can follow the migration phases of FirebaseUI
to change the import(import Firebase compat library when we’re at phase 1, or wait for phase 2 and use import by modules using Firebase v9).

Reply:
Yeah I agree with @jamesdaniels (let's not communicate any plans yet). Let's just focus on informing customers that v6.0.0 should be used with firebase version v9 (compat)and higher. We can also explain some of the changes needed when switching from v8 to v9 (such as the difference in how app and auth are imported or included via cdn, etc.) We can also mention the benefit when using firebaseui v6.0.0 with other v9 firebase services, such as firestore (when it comes to modularization). The description below seems to cover most of that.